### PR TITLE
:rotating_light: feat(alertmanager): add slack integration to alertmanager config file (#11)

### PR DESCRIPTION
**Pull request by niyushabaghayi**
Dear @mehranasghari, the header image should include an icon for Slack next to an icon for Discord, please change the image and push it here.

Dears, @mostafaghadimi and @iamdorsa, I would appreciate your comments.

Thanks, guys.

### CODE REVIEW

File: config_files/alertmanager/config.yaml [hunk 1]
```diff
@@ -3,10 +3,14 @@ route:
   group_wait: 30s
   group_interval: 5m
   repeat_interval: 3h
-  receiver: discord
+  receiver: alerts
 
 receivers:
-- name: discord
+- name: alerts
```
Comment (mostafaghadimi) [inline config_files/alertmanager/config.yaml:9]: Why do we have this name changed?

**Review by mostafaghadimi — COMMENTED** [review]
Well done!

File: config_files/alertmanager/config.yaml (hunk 1 above)
Comment (niyushabaghayi) [reply config_files/alertmanager/config.yaml:9]: According to this part:
```
receivers:
- name: alerts
  discord_configs:
  - webhook_url: <your_discord_webhook>
  slack_configs:
  - send_resolved: true
    text: "{{ .CommonAnnotations.description }}"
    channel: <#your_slack_channel_name>
    api_url: <your_slack_webhook>
    ```
  Now we have two receivers and in the configuration, you can only put one name as the `receiver` in `route` so I change it to a general name.

**Review by mostafaghadimi — APPROVED** [review]
well done guys!

File: config_files/alertmanager/config.yaml (hunk 1 above)
Comment (mostafaghadimi) [reply config_files/alertmanager/config.yaml:9]: Yes, you are right. Thanks!

**Review by mostafaghadimi — APPROVED** [review]

**Pull request merged**